app/src/main/java/com/base_android_template/model/response/GithubUserResponse.kt

```diff
@@ -0,0 +1,59 @@
+package com.base_android_template.model.response
+
+import com.google.gson.annotations.SerializedName
+
+data class GithubUserResponse(
+    @field:SerializedName("login")
+    val login: String,
+
+    @field:SerializedName("id")
+    val id: Int,
+
+    @field:SerializedName("node_id")
+    val node_id: String,
```

Comment:
@LoredanaZdranc I think this is auto-generated. Please remove the _ and rename to `nodeId`